tools/mo/openvino/tools/mo/utils/utils.py

```diff
@@ -25,6 +25,22 @@ def refer_to_faq_msg(question_num: int):
         # Telemetry can be not initialized if it is used in MO IR Reader
         pass
 
+    # TODO: workaround for moved FAQ numbering. But we sent to Telemetry the old wrong numbers,
+    # because telemetry relies on a error messages not on a FAQ text. Changing Telemetry numbers will cause
+    # significant confusion.
+
+    map_to_correct_faq_numbers = {
+        94: 95,  # but actual info in FAQ should be updated
+        97: 93,
+        100: 96,
+        101: 97,
+    }
+
+    if 22 >= question_num <= 93:  # From 22 to 93 they are shifted by one
+        question_num -= 1
+    elif question_num in map_to_correct_faq_numbers:
+        question_num = map_to_correct_faq_numbers[question_num]
```

Comment:
may be it makes sense to update numbers in FAQ?

Reply:
Reverted code changes and updated docs. No committee approval needed for merge into 2023.0

There are blank questions, they are necessary to keep numbering aligned. In the code blank questions were never referred with `refer_to_faq_msg`. Numbering was broken here
https://github.com/openvinotoolkit/openvino/pull/11605 mainly current PR restores that broken numbering.